examples/oop/data_driven_physics.cpp

```diff
@@ -120,10 +121,13 @@ TTS_CASE("Run a ball simulation")
 
     // Display 200 balls
     for(std::size_t i=0;i<200;i++)
```

Comment:
just up to the ball size would be simpler but w/e

Reply:
if you put 10000 balls , it doesn't fit the screen. I just plot the 200 first